### ISSUE TITLE
Help Button on Public Search Issue

### ISSUE THREAD

**Issue by DavidLemayian** [documentcloud/documentcloud#180]
When on the [public search workspace](https://www.documentcloud.org/public/search/), clicking the "Help" button resolves to https://www.documentcloud.org/public/help showing the first help's page without any navigation options. A reload then throws a 404.

**Comment by knowtheory** [comment]
Yep.  Also: https://github.com/documentcloud/documentcloud/issues/171

**Comment by DavidLemayian** [comment]
@knowtheory Yip. Might be a routes + backbone fix?
